components/monitors/cu_consolemon/src/lib.rs

```diff
@@ -282,18 +302,223 @@ struct UI {
     sysinfo: String,
     task_stats: Arc<Mutex<TaskStats>>,
     nodes_scrollable_widget_state: NodesScrollableWidgetState,
+    #[cfg(feature = "debug_pane")]
+    error_redirect: gag::BufferRedirect,
+    #[cfg(feature = "debug_pane")]
+    debug_output: DebugLog,
+}
+
+#[cfg(feature = "debug_pane")]
+struct DebugLog {
+    debug_log: VecDeque<String>,
+    max_rows: Arc<Mutex<u16>>,
```

Comment:
can we get away with a simple Mutex as we only protect a u16? Arc is usually when you want to ship the reference around. Even better an AtomicU16 :)